p/scripts/preview.js

```diff
@@ -20,16 +20,16 @@ function update_ui() {
 
 function init_afterDOM() {
 	rendered_node = document.getElementById('freshrss_rendered');
-	rendered_view = document.getElementById("freshrss_rendered_view");
+	rendered_view = document.getElementById('freshrss_rendered_view');
 
-	raw_node = document.getElementById("freshrss_raw");
-	raw_view = document.getElementById("freshrss_raw_view");
+	raw_node = document.getElementById('freshrss_raw');
+	raw_view = document.getElementById('freshrss_raw_view');
 
 	rendered_node.addEventListener('click', function (ev) {
 		update_ui();
 	});
 
-	raw_node.addEventListener("click", function (ev) {
+	raw_node.addEventListener('click', function (ev) {
 		update_ui();
```

Comment:
Is it actually necessary for this to be an anonymous function?

Reply:
That's a… good question ? 🤷‍♂️

Reply:
In that case, I'd say simplify it to `'click', update_ui` (careful, no `()`) and see if it works. Otherwise that's a lot of wasted lines. ;-)